src/fides/lib/oauth/api/routes/user_endpoints.py

```diff
@@ -241,6 +246,7 @@ def perform_login(
             client_id_byte_length,
             client_secret_btye_length,
             scopes=user.permissions.scopes,  # type: ignore
+            roles=user.permissions.roles,  # type: ignore
```

Comment:
Transferring both scopes and roles from the user's Permission's object to the user's ClientDetail on login.  The token which is later created from this client then has both scopes and roles.